src/test/java/games/strategy/triplea/delegate/MoveDelegateTest.java

```diff
@@ -48,14 +48,14 @@ public void setUp() throws Exception {
     delegate.start();
   }
 
-  private static Collection<Unit> getUnits(final IntegerMap<UnitType> units, final Territory from) {
-    final Iterator<UnitType> iter = units.keySet().iterator();
-    final Collection<Unit> rVal = new ArrayList<>(units.totalValues());
+  private static Collection<Unit> getUnits(final IntegerMap<UnitType> unitCountsByType, final Territory from) {
```

Comment:
The `IntegerMap` parameter was not violating any Checkstyle rule.  However, the most meaningful name for `rVal` was `units`, and thus it conflicted.  So I renamed this parameter to something a bit more descriptive to facilitate the violation fix.

There were several similar changes in other files contained in this PR.